Pear/Controller/ChatViewController.swift

```diff
@@ -281,6 +293,40 @@ class ChatViewController: UIViewController {
             make.height.equalTo(200)
         }
     }
+    
+    @objc private func dismissMenu() {
+        if !displayMenu {
+            optionsMenuView?.removeFromSuperview()
+            displayMenu.toggle()
+        }
+    }
+    
+    @objc private func toggleMenu() {
+        if displayMenu {
+            guard let superView = navigationController?.view else { return }
+            let options = ["Block user"]
+            optionsMenuView = OptionsView(
+                feedbackDelegate: nil,
+                blockDelegate: nil,
+                matchId: messageUser.id,
+                blockId: messageUser.id,
+                options: options,
+                superView: superView
+            )
+            
+            guard let optionsMenuView = optionsMenuView else { return }
+            view.addSubview(optionsMenuView)
+
+            optionsMenuView.snp.makeConstraints { make in
+                make.top.equalTo(menuButton.snp.bottom).offset(8)
+                make.trailing.equalTo(view.snp.trailing).offset(-25)
```

Comment:
`inset(25)` instead of offsetting by a negative number. Also, factor out the constants into variables. you can put them on line 305.